packages/garbo/src/index.ts

```diff
@@ -530,6 +531,11 @@ export function main(argString = ""): void {
       stashItems.push($item`origami pasties`);
     }
 
+    // Prepare Daily Affirmation for PvP fights if desired
+    if (shouldAffirmationHate()) {
+      retrieveItem($item`Daily Affirmation: Keep Free Hate in your Heart`);
```

Comment:
Can you add a price check here?

Reply:
`shouldAffirmationHate` should already cover that